### ISSUE TITLE
QAEval Version Problems

### ISSUE THREAD

**Issue by bowphs** [danieldeutsch/sacrerouge#128]
Hi,
thank you very much for making all these evaluation metrics available!
Sadly, I'm still not able to use QAEval. I tried several options, but did not succeed.
Here is the most promising one:
- python version 3.6.13, sacrerouge==0.2.3, qaeval==0.1.0 (I also tried sacrerouge==0.2.1 and qaeval==0.0.5 as in [this](https://colab.research.google.com/drive/1RikOFUEx299c8qxd6IfCLe3KeuLX31I4?usp=sharing#scrollTo=vPbrTfwN1orD) notebook, but this made no difference.) 
- getting the "zipfile.BadZipFile: File is not a zip file" error mentioned in other issues; solved by manually downloading and copying it into the appropriate directory
- getting the "OSError: [E050] Can't find model 'en_core_web_sm'. It doesn't seem to be a shortcut link, a Python package or a valid path to a data directory." error; solved by entering `python3 -m spacy download en_core_web_sm`
- getting another very unspecific error:
    
```
Traceback (most recent call last):
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/tarfile.py", line 1645, in gzopen
    t = cls.taropen(name, mode, fileobj, **kwargs)
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/tarfile.py", line 1621, in taropen
    return cls(name, mode, fileobj, **kwargs)
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/tarfile.py", line 1484, in __init__
    self.firstmember = self.next()
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/tarfile.py", line 2299, in next
    tarinfo = self.tarinfo.fromtarfile(self)
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/tarfile.py", line 1092, in fromtarfile
    buf = tarfile.fileobj.read(BLOCKSIZE)
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/gzip.py", line 276, in read
    return self._buffer.read(size)
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/_compression.py", line 68, in readinto
    data = self.read(len(byte_view))
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/gzip.py", line 463, in read
    if not self._read_gzip_header():
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/gzip.py", line 411, in _read_gzip_header
    raise OSError('Not a gzipped file (%r)' % magic)
OSError: Not a gzipped file (b'<!')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/site-packages/sacrerouge/metrics/qaeval.py", line 66, in __init__
    batch_size=generation_batch_size, silent=not verbose)
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/site-packages/qaeval/generation/model.py", line 31, in __init__
    self.predictor = Predictor.from_path(model_path, predictor_name='question_generation', cuda_device=cuda_device)
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/site-packages/allennlp/predictors/predictor.py", line 277, in from_path
    load_archive(archive_path, cuda_device=cuda_device),
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/site-packages/allennlp/models/archival.py", line 167, in load_archive
    with tarfile.open(resolved_archive_file, "r:gz") as archive:
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/tarfile.py", line 1591, in open
    return func(name, filemode, fileobj, **kwargs)
  File "/home/freddy/anaconda3/envs/repro/lib/python3.6/tarfile.py", line 1649, in gzopen
    raise ReadError("not a gzip file")
tarfile.ReadError: not a gzip file
```

Could you please help me with this? Thank you very much in advance.

**Comment by bowphs** [comment]
Update:
Solved the issue by downloading the corresponding tarball from [here](https://drive.google.com/u/0/uc?id=1vVhRgLtsQDAOmxYhY5PMPnxxHUyCOdQU&export=download) and inserting it into `blablaba/models/generation/`.

Had some additional troubles with the right spacy version and with my laptop having no NVIDIA GPU, but I did manage to get scores.

**Comment by danieldeutsch** [comment]
Thanks for reporting this. I believe it's the same error as in #120 which will soon be fixed (#129). Also, if you have access to a machine with Docker, [this](https://repro.readthedocs.io/en/latest/models/deutsch2021.html) implementation should be much easier to use.
